spec/models/queen_spec.rb

```diff
@@ -4,4 +4,46 @@
   it "should have a valid factory" do
     expect(create(:queen)).to be_valid
   end
+
+  describe "valid_move?" do
+    
+    let(:game) { create(:game)}
+    let(:queen) { create(:queen, x_pos:5, y_pos:5, game: game) }
```

Comment:
If you create a queen, the factory automatically hooks up a game that is associated with it. So you don't need the `let(:game) { create(:game) }`